hive/src/main/java/org/apache/iceberg/hive/CustomHiveTableOperations.java

```diff
@@ -90,6 +90,11 @@ protected void doRefresh() {
           String errMsg = String.format("%s.%s is missing %s property", database, tableName, METADATA_LOCATION_PROP);
           throw new IllegalArgumentException(errMsg);
         }
+        if (!io().newInputFile(metadataLocation).exists()) {
+          String errMsg = String.format("%s.%s has non existent %s property",
```

Comment:
The error message seems to not match the error here. Error is that the property exists but the file described in the property does not exists. Error message seems to indicate that the property is not set on the table. Can we throw a better error message? Preferably should have the metadata location included somewhere in the error message.